src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -565,7 +565,7 @@ private function getFiltersParameters(CollectionOperationInterface|HttpOperation
             }
 
             foreach ($filter->getDescription($entityClass) as $name => $data) {
-                $schema = $data['schema'] ?? (\in_array($data['type'], Type::$builtinTypes, true) ? $this->jsonSchemaTypeFactory->getType(new Type($data['type'], false, null, $data['is_collection'] ?? false)) : ['type' => 'string']);
+                $schema = $data['schema'] ?? (\in_array($data['type'] ?? false, Type::$builtinTypes, true) ? $this->jsonSchemaTypeFactory->getType(new Type($data['type'], false, null, $data['is_collection'] ?? false)) : ['type' => 'string']);
```

Comment:
specify a `type` in your filter description instead?

Reply:
Sorry for the confusion, but I was just trying to add some DX, instead of having a PHP error when you forget the `type` key.
In that case I fixed, it will return `['type' => 'string']` if there's no type.